src/app/_services/auth-provider.tsx

```diff
@@ -50,14 +43,14 @@ export function AuthProvider({ children }: { children: ReactNode }) {
   const login = (newToken: string) => {
     localStorage.setItem('token', newToken);
     setToken(newToken);
+    queryClient.invalidateQueries({ queryKey: ['myProfile'] });
```

Comment:
@eric5025 @yihimin 
queryKey를 저렇게 별도 참조 없이 사용하게 되면 나중에 관리가 어려워질수 있어요
아래 방법처럼 queryOptions라는것을 tanstack/react-query에서 지원해주니 한번 검토해보셔요

```ts
// src/app/queries/profile.ts
import { queryOptions } from '@tanstack/react-query';

export const profileQueries = {
   myProfile: () => queryOptions({ 
        queryKey: ['myProfile'],
         queryFn: () => getMyProfile();
   })
}
```

```suggestion
queryClient.invalidateQueries(profileQueries.myProfile());
```

Reply:
네!  좀 더 알아보고 리뷰 반영해보겠습니다

Reply:
https://github.com/modern-agile-team/9term-main-front/pull/56#discussion_r2161151497

@yihimin @eric5025 
이전에 @eric5025 님께 비슷한 코멘트를 남겼었고, 이쪽 코멘트와 비슷한 맥락입니다 queryOptions도 좋은 것 같네요